### PR TITLE
Changes to Docker instructions in Readme

### PR DESCRIPTION
**Pull request by ilario**
For me the current version of the command in the readme didn't work, this works.
Can anyone confirm?
And the `-it` option doesn't seem to  be needed.

### CODE REVIEW

**Comment by nicopace** [comment]
This is what -it does:

```
       -i, --interactive=true|false
          Keep STDIN open even if not attached. The default is false.

       When set to true, keep stdin open even if not attached. The default is false.

       -t, --tty=true|false
          Allocate a pseudo-TTY. The default is false.

       When set to true Docker can allocate a pseudo-tty and attach to the standard input of any container. This can be used, for example, to run a throwaway interactive shell. The default is false.

       The -t option is incompatible with a redirection of the docker client standard input.
```

**Comment by ilario** [comment]
Thanks. So it's not needed for current version of cooker, do you agree? @nicopace

**Comment by nicopace** [comment]
Without -i you wll not have an interactive console... ie, not able to run bash. if you run cooker directly, then it is not needed.

Don't know about -t.

File: README.md [hunk 1]
```diff
@@ -27,8 +27,9 @@ For other systems, you might follow these instructions (look for _Examples of Pa
 
 Install [Docker](https://www.docker.com/get-docker) and run the following command:
 
-	docker build -t cooker .
-	docker run -it cooker -v "$(pwd)":/app --<parameters>
+	cd lime-sdk
+	sudo docker build -t cooker .
+	sudo docker run -v "$(pwd)":/app cooker --<parameters>
```
Comment (aparcar) [inline README.md:32]: the `-it` puts all `stdin` to console and you can interact with it, like pressing `y` to continue or whatever
I'm not 100% sure but i think both are required

**Review by aparcar — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ilario) [reply README.md:32]: nowadays nor cooker nor LEDE build system used by cooker accept any interactive input, if cooker will get interactive in the future we can put that option back :)

**Comment by ilario** [comment]
I just realized a thing where `-it` is really useful: Ctrl + C for interrupting easily the compilation.

**Comment by nicopace** [comment]
> I just realized a thing where -it is really useful: Ctrl + C for interrupting easily the compilation.

Should we revert then?

**Comment by ilario** [comment]
> Should we revert then?

Nope, sorry, in this PR there were three changes and removing `-it` was just one of them. I'll do another PR...

**Pull request merged**